Autocoders/Python/src/fprime_ac/generators/templates/enums/enum_hpp.tmpl

```diff
@@ -96,6 +96,11 @@ namespace ${namespace} {
         const ${name}& other //!< The other object
         );
 
+    //! Assignment operator
+    ${name}& operator=(
+        const t a //!< The integer to copy
```

Comment:
This is not an integer; it's an enumerated constant.

Reply:
Good catch.